File: tests/host-streams/oracle.mjs
import {stream} from './library.mjs';
for await(const value of stream('Kernel::Numbers',[3]))console.log(value.toString());
for await(const value of stream('Kernel::Text'))console.log(value??'');
for await(const value of stream('Kernel::Flags'))console.log(value?'True':'False');

File: tests/host-streams/oracle.py
import asyncio
from library import stream
async def main():
 async for value in stream('Kernel::Numbers',[3]):print(value)
 async for value in stream('Kernel::Text'):print('' if value is None else value)
 async for value in stream('Kernel::Flags'):print(value)
asyncio.run(main())
